docs/proposals/loadbalancer-proposal.md

```diff
@@ -161,6 +162,16 @@ Further, we can eventually introduce the 'request/limit model' for network resou
 to acheieve functionalities already implemented in compute resource, for example,
 qos and overcommit.
 
+##### Manually assign LoadBalancerClaim to a LodBalancer
```

Comment:
In previous paragraph, we mentioned one can bind a LoadBalancer directly, maybe we should rephrase.
> change existing Ingress API to include a LBSource field, which either directly uses a LoadBalancer, or via a LoadBalancerClaim (think of Pod using PV or PVC)